### PR TITLE
ostree: follow redirect while getting remote URLs

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

File: src/rootfstreemanager.cc [hunk 1]
```diff
@@ -60,7 +60,7 @@ void RootfsTreeManager::installNotify(const Uptane::Target& target) {
 }
 
 void RootfsTreeManager::getAdditionalRemotes(std::vector<Remote>& remotes, const std::string& target_name) {
-  const auto resp = http_client_->post(gateway_url_ + "/download-urls", Json::Value::null);
+  const auto resp = http_client_->post(gateway_url_ + "/download-urls", Json::Value::null, true);
```
Comment (pattivacek) [inline src/rootfstreemanager.cc:63]: Why not make following redirects the default? I think we already have a limit in place for the number of redirects. What's the risk?

Comment (mike-sul) [reply src/rootfstreemanager.cc:63]: TBH, I don't know. I just wanted to make sure that I won't break anything else. 
Let's try and enable it by default. The question is: Should it be default for any request (set in ctor) or default specifically for POST?

Comment (pattivacek) [reply src/rootfstreemanager.cc:63]: I'd say in can be in the ctor, as long as the setting to limit the number of redirects is also set there. I think redirects are generally safe, as long as there aren't thousands of them.

**Comment by mike-sul** [comment]
Closing this PR in favor of https://github.com/foundriesio/aktualizr-lite/pull/164

**Pull request closed without merging**